### PR TITLE
[QT-426] Add support for enabling the file audit device for enos scenarios

### PR DESCRIPTION
**Pull request by mikebaum**
- Updated the `vault_cluster` enos module to add support for enabling the file audit device. 
- Updated the all scenarios to add support for enabling the file audit device.

### CODE REVIEW

File: enos/modules/vault_cluster/variables.tf [hunk 1]
```diff
@@ -174,3 +174,9 @@ variable "unseal_method" {
     error_message = "The unseal_method must be either awskms or shamir. No other unseal methods are supported."
   }
 }
+
+variable "enable_file_audit_device" {
+  description = "If true the file audit device will be enabled at the path /var/log/vault/vault_audit.log"
+  type        = bool
+  default     = false
```
Comment (jaymalasinha) [inline enos/modules/vault_cluster/variables.tf:181]: why is the variable default here, different than the default in `enos-variables.hcl` ?

Comment (mikebaum) [reply enos/modules/vault_cluster/variables.tf:181]: Good point, I'll change the default to be true.

**Review by jaymalasinha — APPROVED** [review]
LGTM 👍 ! Just one minor comment, may be we should allow the `audit_device_file_path` also be configurable through ENOS_VAR and set the default path with it ?

File: enos/modules/vault_cluster/scripts/enable_audit_logging.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#!/bin/env sh
+
+set -eux
+
+LOG_DIR=$(dirname "$LOG_FILE_PATH")
+
+# setup dir
+if [ ! -d "$LOG_DIR" ]
+then
+  sudo mkdir -p "$LOG_DIR"
```
Comment (jaymalasinha) [inline enos/modules/vault_cluster/scripts/enable_audit_logging.sh:10]: can avoid repeating the permission here with `-R`

File: enos/modules/vault_cluster/variables.tf [hunk 3]
```diff
@@ -174,3 +174,9 @@ variable "unseal_method" {
     error_message = "The unseal_method must be either awskms or shamir. No other unseal methods are supported."
   }
 }
+
+variable "enable_file_audit_device" {
+  description = "If true the file audit device will be enabled at the path /var/log/vault/vault_audit.log"
```
Comment (jaymalasinha) [inline enos/modules/vault_cluster/variables.tf:179]: need to update path description here

Comment (mikebaum) [reply enos/modules/vault_cluster/variables.tf:179]: yeah I caught that... should be updated now.

**Comment by mikebaum** [comment]
> LGTM 👍 ! Just one minor comment, may be we should allow the `audit_device_file_path` also be configurable through ENOS_VAR and set the default path with it ?

I like the idea, but considering the issues I encountered with permissions I'd rather not let the users pick a directory. Maybe we can do that later, if requested. The path has been added to the outputs, plus it's possible to get the path via a CLI command.

**Pull request merged**